src/mixer/main.go

```diff
@@ -224,6 +226,69 @@ func cmdAddBundles(args []string) {
 	b.AddBundles(bundles, *force, *git)
 }
 
+func cmdInit(args []string) {
+	initcmd := flag.NewFlagSet("init", flag.ExitOnError)
+	confOnlyflag := initcmd.Bool("config-only", false, "Only initialize the builder config")
+	allflag := initcmd.Bool("all", false, "Create a mix with all Clear bundles included")
+	clearflag := initcmd.Int("clearver", 0, "Supply the Clear version to compose the mix from")
+	mixflag := initcmd.Int("mixver", 0, "Supply the Mix version to build")
+	initcmd.Parse(args)
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	err = os.Mkdir(pwd+"/rpms", 0755)
+	if os.IsExist(err) {
+		fmt.Println("Directory '/rpms' already exists. Skipping...")
+	} else if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	err = os.Mkdir(pwd+"/local", 0755)
+	if os.IsExist(err) {
+		fmt.Println("Directory '/local' already exists. Skipping...")
+	} else if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	err = helpers.CopyFile(pwd+"/builder.conf", "/usr/share/defaults/bundle-chroot-builder/builder.conf", false)
+	if os.IsExist(err) {
+		fmt.Println("File 'builder.conf' already exists. Skipping...")
+	} else if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	} else {
+		raw, err := ioutil.ReadFile(pwd + "/builder.conf")
+		if err != nil {
+			helpers.PrintError(err)
+			os.Exit(1)
+		}
+
+		data := strings.Replace(string(raw), "/home/clr/mix", pwd, -1)
+		data += "\nRPMDIR=" + pwd + "/rpms\n"
+		data += "REPODIR=" + pwd + "/local\n"
+
+		re := regexp.MustCompile("<(.)+>\n")
+		data = re.ReplaceAllString(data, "file://"+pwd+"/update/www\n")
+
+		err = ioutil.WriteFile(pwd+"/builder.conf", []byte(data), 0666)
+	}
+
+	if *confOnlyflag {
+		return
+	}
+
+	b := builder.New()
+	b.LoadBuilderConf(pwd + "/builder.conf")
+	b.ReadBuilderConf()
+	b.InitMix(strconv.Itoa(*clearflag), strconv.Itoa(*mixflag), *allflag)
```

Comment:
Up for discussion, but I would consider just merging this functionality with the init-mix command, rather than wrapping it/duplicating all of its parameters.

Reply:
Hmm, I think ideally I would have a third separate function that is called by both.

Reply:
I also do not agree with duplicating the command like this. Can this be reworked and be its own separate thing? I wouldn't be against it replacing the current one even...

Reply:
@tmarcu do you mean dropping init-mix and keeping init only?

Reply:
@rchiossi yeah..it essentially looks like the exact same code but with nicer dir creation logic and config-only options :) Doesn't seem like it'd interfere with anyone currently using mixer init-mix. I'd argue to just replace the current one and replace it with your implementation, however keep the call init-mix because people currently using mixer may have it scripted/automated to call "init-mix" and not "init" so we don't want to break them right now. Can update that in a separate PR and send out a disclaimer email along with it to make sure everyone moves over smoothly over time. Another option would be to support both options and call the same code for either.